assets/queries/cloudFormation/ec2_network_ACL_deny_rules/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::EC2::NetworkAclEntry"
+  action := resource.Properties.RuleAction
+  cidr := resource.Properties.CidrBlock
+  unvalidCidr := "0.0.0.0/0"
+  not contains(cidr,unvalidCidr)
+  expectedAction := "deny"
+  contains(action,expectedAction) 
```

Comment:
no need to declare a variable for this constant value: `contains(action,"deny")`